app/controllers/timeline_controller.rb

```diff
@@ -51,7 +51,7 @@ def set_breadcrumbs
   end
 
   def calendar
-    @calendar ||= @timelineable.national_calendar
+    @calendar ||= @timelineable.national_calendar || Calendar.default_national
```

Comment:
I think this is OK for the timeline page. The scoreboards will use a different default period I think.